spec/acceptance/place_order_spec.rb

```diff
@@ -126,41 +126,56 @@
 
   context 'given valid item detais' do
     it 'stores the item details' do
-      items_details = [
-        { part_id: '123', part_name: 'Bits', part_price: '5.00', part_quantity: '1'}
-      ]
-      save_items_details.execute(items_details: items_details)
-      items = items_gateway.all
-      expect(items).to eq(items_details)
+      items = [{ 
+        id: '123',
+        name: 'Bits', 
+        price: '5.00',
+        quantity: '1'
+      }]
+      save_items_details.execute(items: items)
```

Comment:
Maybe rename the hard coded hash to order (or maybe ordered items). 

Then rename the stuff we get back from the gateway below to items.